src/main/java/co/cask/gcp/bigquery/BigQuerySourceConfig.java

```diff
@@ -19,12 +19,12 @@
 import co.cask.cdap.api.annotation.Description;
 import co.cask.cdap.api.annotation.Macro;
 import co.cask.cdap.api.annotation.Name;
-import co.cask.gcp.common.GCPConfig;
+import co.cask.gcp.publisher.GCPReferenceConfig;
 
 /**
  * Class description here.
  */
-public final class BigQuerySourceConfig extends GCPConfig {
+public final class BigQuerySourceConfig extends GCPReferenceConfig {
```

Comment:
GCPReferenceConfig has 'source/sink' in its description, which is bad. Can you create a source config and a sink config where the source config says 'source' and the sink config says 'sink'.